### DISCUSSION TITLE
Keystatic is reformatting my content, how can I stop it?

### DISCUSSION THREAD

**Discussion by vivgui** [Thinkmill/keystatic#917] [Q&A] [answered]
I have a markdown file with this structure
```markdown
- **MacBook Pro 14" M1 Pro, 32GB Ram, 512GB SSD** \
My main machine, this is the fastest laptop I have ever owned
- [**Dell Ultrawide U4919DW**](https://amzn.to/) \
The ultimate productivity monitor
- [**iPad Mini 6th Generation**](https://amzn.to/) \
For reading and media consumption
```
which gets compiled to this
```html
<li><strong>MacBook Pro 14" M1 Pro, 32GB Ram, 512GB SSD</strong> <br>My main machine, this is the fastest laptop I have ever owned</li>
```
I renamed it `mdoc`  to use keystatic on my project but now when I add a new list item/line whatever, the document gets reformatted and recompiled to this
```html
<li><strong>MacBook Pro 14" M1 Pro, 32GB Ram, 512GB SSD</strong> My main machine, this is the fastest laptop I have ever owned</li>
```
And I lose the line break (the <br> tag)

It seems this is happening only when I update the post, I see on my console it happens after this api call

```[200] /api/keystatic/blob/95781986f59fb0e2a8f7cdb607d59adcf4ff2ace/src/content/posts/uses.mdoc```

Is there a way to stop this from happening?

**Comment by emmatown** [comment, marked as answer]
This has been fixed in #918 and published in `@keystatic/core@0.3.9`

**Comment by vivgui** [comment]
@emmatown thank you so much for such a quick fix!
